Sources/Valet.swift

```diff
@@ -67,33 +67,31 @@ public final class Valet: NSObject, KeychainQueryConvertible {
     // MARK: Public Class Methods
     
     /// - parameter identifier: A non-empty string that uniquely identifies a Valet.
-    /// - parameter flavor: A description of the Valet's capabilities.
-    /// - returns: A Valet that reads/writes keychain elements with the desired flavor.
-    public class func valet(with identifier: Identifier, flavor: Flavor) -> Valet {
-        let key = Service.standard(identifier, .valet(flavor)).description as NSString
-        if let existingValet = identifierToValetMap.object(forKey: key) {
-            return existingValet
-            
-        } else {
-            let valet = Valet(identifier: identifier, flavor: flavor)
-            identifierToValetMap.setObject(valet, forKey: key)
-            return valet
-        }
+    /// - parameter accessibility: The desired accessibility for the Valet.
+    /// - returns: A Valet that reads/writes keychain elements with the desired accessibility and identifier.
+    public class func valet(with identifier: Identifier, accessibility: Accessibility) -> Valet {
+        return findOrCreate(identifier, flavor: Flavor.vanilla(accessibility))
+    }
+
+    /// - parameter identifier: A non-empty string that uniquely identifies a Valet.
```

Comment:
"a Valet" -> "a Valet that synchronizes data with iCloud"? (I know we didn't do that before, so nbd)